app/utils/eeg/muse.ts

```diff
@@ -1,18 +1,31 @@
 import 'hazardous';
 import { withLatestFrom, share, startWith, filter } from 'rxjs/operators';
-import { addInfo, epoch, bandpassFilter, addSignalQuality } from '@neurosity/pipes';
+import {
+  addInfo,
+  epoch,
+  bandpassFilter,
+  addSignalQuality,
+} from '@neurosity/pipes';
 import { release } from 'os';
 import { MUSE_SERVICE, MuseClient, zipSamples } from 'muse-js';
 import { from } from 'rxjs';
 import { parseMuseSignalQuality } from './pipes';
-import { MUSE_SAMPLING_RATE, MUSE_CHANNELS, PLOTTING_INTERVAL } from '../../constants/constants';
+import {
+  MUSE_SAMPLING_RATE,
+  MUSE_CHANNELS,
+  PLOTTING_INTERVAL,
+} from '../../constants/constants';
 
 const INTER_SAMPLE_INTERVAL = -(1 / 256) * 1000;
 
 let bluetooth = {};
-let client = {};
-if (process.platform !== 'win32' || release().split('.')[0] >= 10) {
+let client: MuseClient;
+if (
+  process.platform !== 'win32' ||
+  parseInt(release().split('.')[0], 10) >= 10
+) {
   // Just returns the client object from Muse JS
+  // eslint-disable-next-line global-require
   bluetooth = require('bleat').webbluetooth;
```

Comment:
I think that since we have now upgraded our version of electron we can stop using bleat for windows bluetooth. However, I'd like to make that change and QA it in a separate PR.